src/modules/tests/tests.selector.ts

```diff
@@ -47,18 +49,18 @@ export const getTestOutcomeText = (test: StandardCarTestCATBSchema) => {
   return TestOutcome.Terminated;
 };
 
-export const isTestOutcomeSet = (test: StandardCarTestCATBSchema) => {
+export const isTestOutcomeSet = (test: CatBUniqueTypes.TestResult) => {
```

Comment:
A common test result with just the common stuff typed here would allow reuse across categories

Reply:
Changed it to use the Common TestData type